### DISCUSSION TITLE
Are there any default "not immediately published fields" in the eForms?

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#636] [Q&A] [answered]
I remember that earlier it was said that not all fields are published immediately (without using the fields of BG-8 Not Immediately Published). I couldn't find any document of this on https://docs.ted.europa.eu/ (only that how to use the fields of BG-8 Not Immediately Published). Are there any default "not immediately published fields" in the eForms/SDK? Can you tell me which fields are these?

My problem is:
1. We generated eForms XML with subtype 25, 28.
2. We generated PDF with Visualisation API.
3. We cannot find in the PDF the next fields:
- BT-161 Notice Value
- BT-118 Notice Framework Maximum Value
- BT-1118 Notice Framework Approximate Value

I have checked it on the live TED: I searched an Estonian notice with 25 subtype and I find the same situation in there. 
That is why I think there are some data which are not published immediately ... or maybe never??

**Comment by pdonohoe** [comment, marked as answer]
Dear AnnaKSZ,
The only mechanism to delay publication of fields requires using BG-8 Not Immediately Published fields. There are no default "not immediately published" fields.
There is a bug in the EFX code for eForms subtypes 25 to 28, which means that no information about the Winner (including the fields you mention) are displayed. This bug will be fixed in a future release of the SDK, and the fix will be applied to previous SDK versions, in new patch versions.
